src/teams.json

```diff
@@ -3,19 +3,45 @@
     {
       "name": "storage",
       "apis": [
-        "redis",
         "storage",
         "storagetransfer"
+      ],
+      "repos": [
+        "googleapis/java-storage",
```

Comment:
You don't need to specifically call these repos out.  All the storage stuff gets auto-flagged with `api: storage` labels, so it happens automagically.  This isn't going to hurt anything, but it does muddy this file up a bit.